src/styles/blocks/day.scss

```
@@ -0,0 +1,27 @@
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  @include square($dayCellSize);
+  margin: 0;
+
+  background-color: #eee;
+  border: 1px solid #000;
+
+  transition-duration: 0.5s;
```

you should specify the properties that will change
```suggestion
  transition-duration: background-color 0.5s, transform 0.5s;
```